packages/nx-plugin/src/generators/configure-storybook/files/.storybook/preview.tsx.template

```diff
@@ -1,18 +1,29 @@
-
-import * as React from 'react';
-
 import { Preview } from '@storybook/react';
+import { withThemeFromJSXProvider } from '@storybook/addon-themes';
 
-import { FluentProvider, webLightTheme } from '@fluentui/react-components';
+import {
+  FluentProvider,
+  webLightTheme,
+  webDarkTheme,
+  teamsLightTheme,
+  teamsDarkTheme,
+  teamsHighContrastTheme,
+} from '@fluentui/react-components';
 
 const preview: Preview = {
   decorators: [
-    (Story) => (
-      <FluentProvider theme={webLightTheme}>
-        <Story />
-      </FluentProvider>
-    ),
+    withThemeFromJSXProvider({
```

Comment:
this starts to become a maintenance burden / hard to manage boilerplate -  this should be implemented as part of sb plugin ( we  have this in core repo AFAIR ? -> it will be published and streamlined here as well in future )

I'd start just with adding this functionality to your package in order to not introduce unmaintained boilerplate while accommodating your package needs. wdyt?

Reply:
I thought about this as well, but after noticed it's not published and was't sure if it's planned to be published, decided to use this approach as for contrib that might be sufficient to use official sb plugin for theming. Will close this one then. Thanks.